src/PresentationalComponents/PathwaysTable/PathwaysTable.cy.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations/';
+import { getStore } from '../../Store';
+import PathwaysTable from './PathwaysTable';
+import fixtures from '../../../cypress/fixtures/pathways.json';
+import { pathwaysTableColumns } from '../../../cypress/support/globals';
+import _ from 'lodash';
+
+// eslint-disable-next-line rulesdir/disallow-fec-relative-imports
+import {
+  TOOLBAR,
+  checkTableHeaders,
+  tableIsSortedBy,
+  checkPaginationTotal,
+  checkPaginationValues,
+  changePagination,
+  PAGINATION_VALUES,
+  SORTING_ORDERS,
+} from '@redhat-cloud-services/frontend-components-utilities';
+
+const ROOT = 'table[aria-label="pathways-table"]';
+const TABLE_HEADERS = _.map(pathwaysTableColumns, (it) => it.title);
+const DEFAULT_ROW_COUNT = 4;
```

Comment:
Default pagination is set to 20, not 4.

Reply:
It's not to check default pagination but the how many rows of data we have.
In the UI it will look like "1-4 of 4". We're targeting the second 4 with this number